website/content/docs/k8s/connect/ingress-controllers.mdx

```diff
@@ -0,0 +1,79 @@
+
+* Enable connect-injection via an annotation on the Ingress Controller's deployment: `consul.hashicorp.com/connect-inject` is `true`.
+
+* Using the following annotations on the Ingress controller's deployment, setup exclusion rules for it's ports.
+  * [`consul.hashicorp.com/transparent-proxy-exclude-inbound-ports`](https://www.consul.io/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-inbound-ports) - Provides the ability to exclude a list of ports for
+inbound traffic that the service exposes from redirection. Typical configurations would require all inbound service ports
+for the controller to be included in this list.
+  * [`consul.hashicorp.com/transparent-proxy-exclude-outbound-ports`](https://www.consul.io/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-outbound-ports) - Provides the ability to exclude a list of ports for
+outbound traffic that the service exposes from redirection.
+  * [`consul.hashicorp.com/transparent-proxy-exclude-outbound-cidrs`](https://www.consul.io/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-outbound-cidrs) - Provides the ability to exclude a list of CIDRs that
+the service communicates with for outbound requests from redirection. It is somewhat common that an Ingress controller
+will expect to make API calls to the Kubernetes service for service/endpoint management. As such including the ClusterIP of the
+Kubernetes service is common.
+
+~> Note: Depending on which ingress controller you use, these stanzas may differ in name and layout, but it is important to apply
+these annotations to the *pods* of your *ingress controller*.
+
+```yaml
+# An example list of pod annotations for an ingress controller.
+  podAnnotations:
+    consul.hashicorp.com/connect-inject: "true"
+    consul.hashicorp.com/transparent-proxy-exclude-inbound-ports: "80,8000,9000,8443"
+    # And the CIDR of your Kubernetes API svc
+    consul.hashicorp.com/transparent-proxy-exclude-outbound-cidrs: "10.108.0.1/32"
+```
+
+* If the Ingress controller acts as a LoadBalancer and routes directly to Pod IPs instead of the ClusterIP of your Kubernetes Services
+a `ServiceDefault` CRD must be applied to *each backend service* allowing it to use the `dialedDirectly` features. By default this is disabled.
+
+```yaml
+# Example Service defaults config entry
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceDefaults
+metadata:
+  name: backend
+spec:
+  transparentProxy:
+    dialedDirectly: true
+```
+
+* An intention from the Ingress Controller to the backend application must also be applied, this could be an L4 or L7 intention:
+
+```yaml
+# example L4 intention, but an L7 intention can also be used to enforce traffic to specific routes.
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceIntentions
+metadata:
+  name: ingress-backend
+spec:
+  destination:
+    name: backend
+  sources:
+    - name: ingress
+      action: allow
+```
```

Comment:
These need to be indented to be under the bullets. Be careful because on git commit there's a hook that will reformat. I think it needs to be indented 3 spaces so it doesn't get reformatted.